Stack/Opc.Ua.Core/Stack/Tcp/ChannelToken.cs

```diff
@@ -73,7 +82,7 @@ public bool Expired
         {
             get
             {
-                if (DateTime.UtcNow > m_createdAt.AddMilliseconds(m_lifetime))
+                if (HiResClock.UtcNow.Ticks -  m_createdAtTick > m_lifetime)
```

Comment:
do not use the conversion to utcnow first, use ticks only directly for calculations

Reply:
reverted